src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/ModuleBuilderImpl.cs

```diff
@@ -399,6 +406,196 @@ private void WriteMethods(List<MethodBuilderImpl> methods, List<GenericTypeParam
             }
         }
 
+        private void AddSymbolInfo(ILGeneratorImpl il, StandaloneSignatureHandle localSignatureHandle, MethodDefinitionHandle methodHandle)
+        {
+            if (il.DocumentToSequencePoints.Count > 0)
```

Comment:
Personal preference. I usually place the "small" branch condition first. I do that because I dislike scolling down pages and then seeing a one line statement and have often forgotten the predicate we are branching on. In this case switching to `il.DocumentToSequencePoints.Count <= 0` is clear we are handling the atypical case immediately.